commit f6a4295b4ca5561f79d2e0cf3798708b61385adc (refs/changes/81/3881/2)
Author: Jason King <jason.king@joyent.com>
Date:   2018-04-26T22:04:46-05:00 (1 year, 5 months ago)
    
    OS-6915 Unused variable broke platform-debug build
    Reviewed by: Robert Mustacchi <rm@joyent.com>
    Approved by: Robert Mustacchi <rm@joyent.com>

diff --git a/usr/src/lib/fm/topo/libtopo/common/topo_subr.c b/usr/src/lib/fm/topo/libtopo/common/topo_subr.c
index 9d68061e11..8d9408dd59 100644
--- a/usr/src/lib/fm/topo/libtopo/common/topo_subr.c
+++ b/usr/src/lib/fm/topo/libtopo/common/topo_subr.c
@@ -561,7 +561,7 @@ static const topo_pgroup_info_t auth_pgroup = {
 void
 topo_pgroup_hcset(tnode_t *node, nvlist_t *auth)
 {
-	int err, ret;
+	int err;
 	char isa[MAXNAMELEN];
 	struct utsname uts;
 	char *prod, *psn, *csn, *server;
